lib/clamp-device.in

```diff
@@ -103,12 +103,14 @@ if [ $KMDUMPLLVM == "1" ]; then
 fi
 
 # select appropriate ROCm-Device-Libs per AMDGPU_TARGET
-if [ $AMDGPU_TARGET == "fiji" ]; then
-  OCLC_ISA_VERSION_LIB="$ROCM_LIB/oclc_isa_version_803.amdgcn.bc"
-elif [ $AMDGPU_TARGET == "hawaii" ]; then
+if [ $AMDGPU_TARGET == "gfx701" ]; then
```

Comment:
after testing with end-to-end programs, changes in this file are actually incorrect. I've fixed them in further commits.